examples/adaq4224_DataCapture.m

```diff
@@ -0,0 +1,53 @@
+%% Script for capturing and displaying a continuous set of samples from a 
+%% connected ADAQ4224 board
+
+% Instantiate the ADAQ4224 Rx system object, and specify the uri in this
+% manner
+rx = adi.ADAQ4224.Rx('uri','ip:analog.local');
+
+rx.SamplesPerFrame = 4096;  % Using values less than 3660 can yield poor 
+% performance, generally
+rx.SampleRate = '2000000';
+rx.SampleAveragingLength = '16';
+
+% The parameter below specifies the number of frames or buffers to capture.
+% Refer to the Streaming section in the documentation if discontinuities
+% are observed in the acquired data.
+rx.FrameCount = 1;
+
+% Only a select few channel groupings yield sensible outputs. Refer to the
+% Limitations section in the documentation for more details
```

Comment:
Does the limitations sections doc needs to be added here as well?

Reply:
This is not required (not a limitation).. will remove it.